bodhi/server/push.py

```diff
@@ -38,6 +57,8 @@
               help='Push updates with a specific request (default: testing,stable)')
 @click.option('--resume', help='Resume one or more previously failed pushes',
               is_flag=True, default=False)
+@click.option('--check-sig-status', help='Check signature status with koji',
+              is_flag=True, default=False)
```

Comment:
I recommend making this default to ```True``` because people are likely to forget to use it. Or maybe rename the flag to ```--dont-check-signatures```? Or maybe even don't bother making it a flag at all and have it always do this?

Reply:
If we do make this a flag, let's add it to the man page at ```docs/user/man_pages/bodhi-push.rst```.